manager.py

```diff
@@ -55,43 +71,121 @@ def configure(self, context):
         self.logger.info('REST Manager set to {}'.format(self._rest_manager))
 
         # fetch component settings
-        self._product_api_url_prefix = \
-            Settings.get("configuration", "product_api_url_prefix",
-                         fallback="https://api.nio.works/v1")
-
-        default = Persistence().load("api_key", default=None)
-        self._instance_api_key = \
-            NIOEnvironment.get_variable('API_KEY', default=default)
-
-        self._instance_id = Settings.get("configuration", "instance_id")
-       
+        self.config_api_url_prefix = \
+            Settings.get("configuration", "config_api_url_prefix",
+                fallback="https://api.nio.works/v1/instance_configurations")
+
+        setting_config_id = Settings.get("configuration", "config_id",
+            fallback=None)
+        self.config_id = Persistence().load("configuration_id",
+                                            default=setting_config_id)
+        setting_config_version_id = \
+            Settings.get("configuration", "config_version_id",
+                fallback=None)
+        self.config_version_id = Persistence().\
```

Comment:
This reads it from Persistence and saves it back, it should assign to _config_version_id

Reply:
Once this setting is persisted, the only way to override it is through the API, I wish there was a better solution, I wonder if we should have a setting in nio.conf set by default to `False` allowing to override with the nio.conf setting